astro/src/content/articles/identity-basics/what-is-oidc.mdx

```diff
@@ -0,0 +1,170 @@
+
+## What Does OIDC Allow?
+
+At a high level, using OIDC lets you delegate user authentication in a standardized fashion. By running an OIDC compliant identity data store, other applications, including custom built apps, open source applications, commercial off the shelf apps, and SaaS solutions, can all use the same authentication solution.
+
+For example, you could have your own application (custom-built), forum software like Discourse (open source), Oracle (COTS), and Zendesk all delegate authentication to your OIDC compliant user identity datastore. You can support both employees and customers using such a solution. In fact, because so many customer applications are web apps, OIDC is a perfect fit for a customer identity and access management (CIAM) system.
+
+## Why Would You Use OIDC?
+
+Similar to other single sign-on software, using OIDC has the following benefits:
+
+* your users have one set of credentials to manage
+* you get a single view into your users, whether they are customers, free users or employees
+* when you add layers of authentication security to your OIDC server, all applications benefit without any effort
+
+OIDC is unique because:
+
+* many modern applications support it
+* it's built for web and mobile applications
+* as it is built on OAuth, OIDC flows can get an access token, useful if you are also using APIs
+
+Let's dive into some of the major pieces of OIDC.
+
+## Users, OIDC Providers and Relying Parties
+
+OIDC has a few important components:
+
+* An end user who is trying to get access to an application using a browser or mobile device.
+* An OpenID provider (OP), which is an OAuth 2.0 compatible authorization server. The OP authenticates users. It can also delegate authentication to other identity stores via OIDC or other protocols.
+* A relying party (RP), which is an application that needs to identify the end user.
+
+An example of an OP is FusionAuth. Examples of RPs include:
+
+* Your custom web or mobile application
+* Commercial SaaS software like SaleForce or Zendesk
+* Open source software like Discourse
+* Cloud software providers like AWS and GCP
+
+Here's an example OIDC flow, using the authorization code flow.
+
+<OIDCFlow/>
+
+For the RP, the end result of an interaction with the OP is a set of tokens which contain information about the user. Let's look at those next.
+
+## Tokens
+
+Because OIDC is built on top of OAuth, an access token is always returned, but an Id token and possibly a refresh token are returned as well. The Id token is the star of the OIDC show, because it contains authentication information, so that's what this article will focus on.
+
+The Id token looks like this:
+
+<ExampleIdToken/>
```

Comment:
<img width="623" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/139158618/0a2e8eef-cbef-4dbc-b92c-c49794460cc8">

Could we add something with more imagery so that people will understand where this information comes from? I think some will just see this and get confused.

<img width="885" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/139158618/afded0c0-d269-4ba2-93e2-0e3c1a6a5f71">